packages/filesystem/src/node/nsfw-watcher/nsfw-filesystem-watcher.spec.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (C) 2018 Ericsson and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as temp from 'temp';
+import * as chai from 'chai';
+import * as fs from 'fs-extra';
+import * as assert from 'assert';
+import * as chaiAsPromised from 'chai-as-promised';
+import URI from "@theia/core/lib/common/uri";
+import { FileUri } from "@theia/core/lib/node";
+import { NsfwFileSystemWatcherServer } from './nsfw-filesystem-watcher';
+import { DidFilesChangedParams } from '../../common/filesystem-watcher-protocol';
+// tslint:disable:no-unused-expression
+
+const expect = chai.expect;
+const track = temp.track();
+
+describe("nsfw-filesystem-watcher", function () {
+
+    let root: URI;
+    let watcherServer: NsfwFileSystemWatcherServer;
+    let watcherId: number;
+
+    this.timeout(10000);
+
+    before(() => {
+        chai.config.showDiff = true;
+        chai.config.includeStack = true;
+        chai.should();
+        chai.use(chaiAsPromised);
+    });
```

Comment:
Correct me if I am wrong, but I think this is absolutely useless since you use `async/await`. Please remove it, and if you see this pattern in other PRs in the future, notify the author too. I have seen this copy/pasted setup so many times.

See #928.

Reply:
You're right, it's a bad copy/paste mistake on my part, I removed it in the latest PR.